spec/models/validations/household_validations_spec.rb

```diff
@@ -277,7 +269,7 @@
   end
 
   describe "#validate_partner_count" do
-    let(:log_date) { Time.zone.local(2023, 4, 1) }
+    let(:startdate) { Time.zone.local(2023, 4, 1) }
```

Comment:
Is this true for any date? If so, we can remove this and have startdate now, if not can we put in a context?

Reply:
I think the validation is only relevant for up to 2024